wiki/ReportTOC.md

```diff
@@ -0,0 +1,34 @@
+  * [Package de test/installation](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#package-de-test-installation)
+    + [Web](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#web)
+      - [Installation avec Docker](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#installation-avec-docker)
+      - [Installation avec Yarn](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#installation-avec-yarn)
+      - [Tests d'intégration](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#tests-d-int-gration)
+    + [Android](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#android)
+      - [GooglePlay](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#googleplay)
+      - [Git](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#git)
+    + [Back-end](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#back-end)
+  * [Organisation du code](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#organisation-du-code)
+    + [Web](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#web-1)
+      - [Structure générale](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#structure-g-n-rale)
+      - [Elm](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#elm)
+        * [Structure du code](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#structure-du-code)
+        * [Points notables](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#points-notables)
+    + [Android](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#android-1)
+      - [Structure générale](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#structure-g-n-rale-1)
+      - [Authentification](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#authentification)
+      - [Datamodel](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#datamodel)
+      - [Home](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#home)
+      - [Network](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#network)
+      - [Poll](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#poll)
+      - [Question](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#question)
+      - [Utils](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#utils)
+    + [Back-end](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#back-end-1)
+  * [Suivi Qualité](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#suivi-qualit-)
+    + [Principe général](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#principe-g-n-ral)
+    + [Liste des contrôles effectués](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#liste-des-contr-les-effectu-s)
+    + [Instructions pour vérifier les protocoles](https://github.com/heig-PRO-b04/java-backend/wiki/Rapport#instructions-pour-v-rifier-les-protocoles)
+
+_Created from [here](https://ecotrust-canada.github.io/markdown-toc/)_
```

Comment:
We do not want this line

Reply:
Yeah, I will delete it for the final version of the report, this was just added here in order to avoid looking for the same site over and over again ;)